lib/evss/letters/letters_response.rb

```diff
@@ -19,7 +19,9 @@ def initialize(status, response = nil)
       end
 
       def metadata
-        super().merge(address: address)
+        meta = super
+        meta[:address] = address if ok?
```

Comment:
I thought the purpose of `meta` was to encapsulate the response from evss to vets-api, in which case, does `address` belong?

I recognize that this question is a bit out of scope for this PR and probably warrants it's own design discussion.